packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx

```diff
@@ -71,7 +71,14 @@ const RenderTmpl: FC<{
   return (
     <StructRender
       struct={tmpl.struct}
-      options={{ ...options, autoScaleMargin: 15 }}
+      options={{
+        ...options,
+        autoScaleMargin: 15,
+        fontsz: 55,
```

Comment:
could we put these into constants, since it looks like these are re-used in several files?

Reply:
Done